packages/runtime/runtime-utils/src/remoteFluidObjectHandle.ts

```diff
@@ -66,6 +66,6 @@ export class RemoteFluidObjectHandle extends FluidHandleBase<FluidObject> {
 	}
 
 	public bind(handle: IFluidHandleInternal): void {
-		handle.attachGraph();
+		fail("RemoteFluidObjectHandle not supported as a bind source");
```

Comment:
Consider adding / updating the comments on this class to describe why this is not possible. That way no one should accidentally create this in the wrong context.

Reply:
Yeah I think I'm gonna deprecate bind in this next release

Reply:
See #24553